MAINTAINERS.yml

```diff
@@ -1534,6 +1534,16 @@ Release Notes:
   labels:
     - "area: Memory Management"
 
+"Drivers: MIPI DBI":
+  status: maintained
+  maintainers:
+    - danieldegrasse
+  files:
+    - drivers/mipi_dbi/
+    - dts/bindings/mipi-dbi/
+  labels:
+    - "area: Display"
```

Comment:
Label Display is already confusing, IMO this should be `Display Controller`.

Reply:
Ok, will update it.

Reply:
You may need someone to create the label.

Reply:
You got it: https://github.com/zephyrproject-rtos/zephyr/labels/area%3A%20Display%20Controller

Reply:
@danieldegrasse Would you also like to add yourself to "Display drivers" collaborators?

Reply:
Sure- I'll do so in a separate PR